helpers/GenerateResourcesAndImage.ps1

```diff
@@ -156,7 +158,7 @@ Function GenerateResourcesAndImage {
         if($Force -eq $true) {
             # Cleanup the resource group if it already exitsted before
             Remove-AzResourceGroup -Name $ResourceGroupName -Force
-            New-AzResourceGroup -Name $ResourceGroupName -Location $AzureLocation
+            New-AzResourceGroup -Name $ResourceGroupName -Location $AzureLocation -Tag $tags
```

Comment:
do we need to check if `$tags` variable exists or is it's ok to pass the empty param?

Reply:
> do we need to check if `$tags` variable exists or is it's ok to pass the empty param?

If $tags is $null, the param Tag is ignored (same with New-AzStorageaccount). But I can double check again, and will confirm it later in the day, with documentation update ;)

Reply:
OK, so I checked and with `New-AzResourceGroup` if tag is $null, it's OK ! But, Fortunately to double-check cause with `New-AzStorageAccount` using tag argument as null, I got this error message : 
`New-AzStorageAccount: Cannot validate argument on parameter 'Tag'. The argument is null. Provide a valid value for the argument, and then try running the command again.`
So I'll fix it by using a condition for this command, and let you know when it's ready ;)

Reply:
It's commited -> https://github.com/actions/virtual-environments/pull/5467/commits/de6ad4b63fd3aba683e427bcdf1c2f59a4d0bb53
I let you check it, and resolve comment it's it's ok for you